pkgs/servers/frr/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, libcap
+, json_c
+, readline
+, net-snmp
+, perl
+, texinfo
+, pkg-config
+, c-ares
+, python3
+, python3Packages
+, libyang
+, flex
+, bison
+, openssl
+, czmq
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "frr";
+  version = "7.5.1";
+
+  src = fetchFromGitHub {
+    owner = "FRRouting";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "0lzsvi3kl9zcd4k04vc3363z9v2yrp7wc8bziv6w9h5fznh2vkxp";
+  };
+
+  buildInputs =
+    [ readline net-snmp c-ares json_c python3 libyang openssl czmq ]
+    ++ lib.optionals stdenv.isLinux [ libcap ];
+
+  nativeBuildInputs = [ pkg-config perl texinfo autoreconfHook python3Packages.sphinx python3Packages.pytest flex bison ];
+
+  configureFlags = [
+    "--sysconfdir=/etc/frr"
+    "--localstatedir=/run/frr"
+    "--sbindir=$(out)/libexec/frr"
+    "--disable-exampledir"
+    "--enable-user=frr"
+    "--enable-group=frr"
+    "--enable-configfile-mask=0640"
+    "--enable-logfile-mask=0640"
+    "--enable-vty-group=frrvty"
+    "--enable-snmp"
+    "--enable-multipath=64"
+  ];
+
+  postInstall = ''
+    substituteInPlace $out/libexec/frr/frr-reload --replace /usr/lib/frr/ $out/libexec/frr/
+  '';
```

Comment:
Can this be put into postPatch somehow?

Reply:
I think it can. Why is postPatch preferred?

Reply:
done.